pages/[id].jsx

```diff
@@ -26,7 +24,7 @@ export default function DetailPage({ fetchedTeamMember }) {
           !
         </h1>
       </main>
-    </div>
+    </Layout>
```

Comment:
We're going to have multiple `main#main-content` elements here, since the `Layout` already includes it.